src/lib.rs

```diff
@@ -78,14 +78,12 @@
 
 #[macro_use]
 extern crate log;
```

Comment:
Similar to serde (see below), we *could* import the `log::{debug, info, warn, error}` macros directly, but for logging I think it's nicer if those calls are global.

Reply:
So you'd just do `use log;` and then `log::warn!`?

Reply:
I'm not sure if that works, but you can also do `use log::warn` and then `warn!(...)`. Macros can now be imported like structs or enums.

Reply:
Fixed.